host_guest/Isaacs_clip/host_guest/README.md

```diff
@@ -0,0 +1,9 @@
+# Host
+
+- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb).
+- Trimertrip host files were created as follows:
+  - Original starting structure (in `CIF` file format, which will be added here once the challenge ends) was converted to a PDB using OpenEye toolkits. Water, a guest, a single Na+ ion, and connectivity records were manually removed. OpenEye toolkit was used to normalize the molecule, add explicit hydrogen, assign partial charges using AM1-BCC and save the resulting molecule to a `MOL2` file. The SYBYL atom types in the `MOL2` file were switched to GAFF2 atom types using antechamber, as the parmchk2 toolkit used afterwards expects GAFF/GAFF2 atom types. The parmchk2 toolkit was used to generate parameters and a `frcmod` file and these files were used in tleap (adding TIP3P water and sodium chloride) to generate amber files. The OpenMM toolkit was used to briefly run some dynamics to generate an amber compatible NetCDF coordinate file with initial velocities. Afterwards, an annealing procedure was used where the structure was heated from 0K to 450K for 30ps, then kept at kept at the high temperature for 5ns, then cooled to room temperature (273K) for 30ps and then kept at this temperature for 10ns. The RMSD of the final Trimertrip structure from the starting crystal structure is 3 Angstroms.
```

Comment:
Comment on where the structure came from (provided by Lyle Isaacs, from a crystal structure -- but we are not releasing that crystal structure).

Reply:
Not released until the challenge ends, correct (ie that CIF file is the crystal structure, right?)

Reply:
Yes, right. I would just explain all of that here.

Reply:
(See remarks I added).